packages/code-du-travail-frontend/src/conventions/ConventionTexte.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { fetchTexte } from "../common/convention.service";
+import Sidebar from "./texte/Sidebar";
+import Content from "./texte/Content";
+import styled from "styled-components";
+import { theme } from "@cdt/ui/";
+
+class ConventionTexte extends React.Component {
+  constructor(props) {
+    super(props);
+    if (props.preloadedTexte) {
```

Comment:
I can't find where this props `preloadedTexte` is used ? Is it still used ?

Reply:
it's in the tests, yes I believe it's used